cves/kernel/CVE-2013-2140.yml

```diff
@@ -151,10 +163,16 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    The vulnerability was originally discovered by Konrad Wilk, who also
+    introduced it when implementing the original function. There is no
+    description of how the vulnerability was discovered, but at that time Wilk
+    was the director of Xen, the project this driver is part of. Being more
+    involved with project requirements may have alerted him to this issue since
+    he wrote the function originally, but this is just speculation.
```

Comment:
I'm not sure if this type of document is the place for speculation. I personally might remove this part of the answer.

Reply:
Yeah I was on the fence about keeping this in. I'll remove it and replace it with a bit more a description of where I looked.